ci-operator/jobs/openshift/assisted-image-service/openshift-assisted-image-service-main-periodics.yaml

```diff
@@ -1,7 +1,7 @@
 periodics:
 - agent: kubernetes
   cluster: build02
-  cron: 47 3 * * *
+  cron: 15 1 * * *
```

Comment:
out of curiosity, is this periodic change done automatically by the `make jobs`?

Reply:
Yes, I think it generates the random time based on this code: https://github.com/openshift/ci-tools/blob/0086dc59e3c3480a00c9aba76f4ba5f44aec992a/pkg/prowgen/prowgen.go#L186

Reply:
Yup it is a generated change